core/internal/testutils/testutils.go

```diff
@@ -42,6 +43,13 @@ func NewRandomInt64() int64 {
 	return id
 }
 
+// NewRandomChainID returns a suitable random chain ID that will not conflict
+// with fixtures
+func NewRandomChainID() *big.Int {
```

Comment:
Might want to be proactive and qualify this with EVM or Int as well, since other chain types have different types, like strings on terra.